server/tests/test_mpesa_routes.py

```
@@ -0,0 +1,284 @@
+        response = client.post(
+            "/api/stk-push",
+            data=json.dumps(data),
+            content_type="application/json",
+        )
+
+        assert response.status_code == 200
+        json_data = response.get_json()
+        assert json_data["success"] is True
+        assert json_data["MerchantRequestID"] == "29115-34620561-1"
+
+    def test_initiate_stk_push_missing_parameters(self, client, session):
```

Ruff (0.13.1)

Unused method argument: `session`

(ARG002)